caddy/main.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2022 The Corazawaf Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	caddycmd "github.com/caddyserver/caddy/v2/cmd"
+
+	_ "github.com/caddyserver/caddy/v2/modules/standard"
+	_ "github.com/corazawaf/coraza-caddy"
+	_ "github.com/jptosso/coraza-pcre"
```

Comment:
I added this, I couldn't find how just `go get` as in CI previously would take effect without it.